shared/reducers/tracker.js

```diff
@@ -119,6 +121,30 @@ function updateUserState (state: TrackerState, action: Action): TrackerState {
     case Constants.onUnfollow: // TODO
       return state
 
+    case Constants.onFollow:
+      return {
+        ...state,
+        lastAction: 'followed',
+        reason: `You have followed ${state.username}.`
```

Comment:
Here's the main new actions -- clicking Follow results in lastAction and reason changing, and the header and action bar pick those up.

Should the new reason come from the service?  Maybe it's okay to set it ourselves.